packages/search-service/src/search-sitemap/schema.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  Document, Model, model, Schema,
+} from 'mongoose';
+
+export const SiteMapSchema: Schema = new Schema({
+  url: { type: String },
+  contentType: { type: String },
```

Comment:
i think lastModified to be in the db

Reply:
Last modified is not needed, as that will be a inferred field and thus should not be stored in the collection.